configure_metrics_agent.sh

```diff
@@ -98,7 +100,7 @@ supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 minfds = 1024
 minprocs = 200
 loglevel = info
-logfile = /var/log/monasca/agent/supervisord.log
+logfile = ${MON_CUSTOM_AGENT_LOG_DIR}/supervisord.log
```

Comment:
I guess it should be ${MON_AGENT_LOG_DIR}

Reply:
Ugh, thx for spotting.
Done.